setup.py

```diff
@@ -28,7 +28,7 @@
 
 INSTALL_REQUIRE = [
     "torch>=1.11",
-    "transformers>=4.36,<4.45",
+    "transformers>=4.36,<4.46",
     "optimum~=1.22",
```

Comment:
should `optimum` version be also updated? As it was there https://github.com/huggingface/optimum-intel/commit/2b621cff00765f88bd15197ddd57ec6c1ef91d07

Otherwise, optimum forces older `transformers` version.

Reply:
that will be done in this PR https://github.com/huggingface/optimum-intel/pull/926